LVT/LVT.Services/BetPresenter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using LVT.LVT.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LVT.LVT.Services
+{
+    public class BetPresenter : IVisualizer<Bet>
+    {
+        public string VisualizeToString(Bet bet, string parentNode)
+        {
+            string result = "[{ v: '" + bet.NodeID + "', f: 'Bet" + "<div style=\"font-style:italic\">" + bet.Title + "</div>'}, " + $"'{parentNode}']";
+
+            if (bet.Initiatives != null && bet.Initiatives.Count >= 1)
```

Comment:
Is there a need for `!= null` and `.Count >= 1`?

Reply:
Assume these comments are there for all ...Presenter.cs classes

Reply:
At the moment there is, because currently all objects (bet, epic, goal etc) can be created without initialising a list.  Could be changed by initialising those objects with empty lists if we wanted to.